tests/karate/src/test/java/fe/net/mockbackend.feature

```diff
@@ -0,0 +1,20 @@
+@ignore @report=false
+Feature: Mock Backend
+
+  Scenario: Setup Mock-backend
+    * def getBackend =
+            """
+              function() {
+                var Backend = Java.type("fe.net.MockBackend")
+                return new Backend(9005)
```

Comment:
Same remark as above the port parameter can be placed in karte-config.js and later 
configured  with `* configure port 9005`